### PR TITLE
APTU: Stubbed NotifyToWait, taken from 3dmoo.

### PR DESCRIPTION
**Pull request by Subv**
Also renamed some handles in the APT:U service to be more descriptive.
Fixed a typo in InquireNotification

### CODE REVIEW

**Comment by purpasmart96** [comment]
@subv You are aware of #345 right?

File: src/core/hle/service/apt_u.cpp [hunk 1]
```diff
@@ -84,7 +98,7 @@ void Enable(Service::Interface* self) {
 
 void InquireNotification(Service::Interface* self) {
     u32* cmd_buff = Kernel::GetCommandBuffer();
-    u32 app_id = cmd_buff[2];
+    u32 app_id = cmd_buff[1];
```
Comment (purpasmart96) [inline src/core/hle/service/apt_u.cpp]: What's this? Why was this changed?

Comment (Subv) [inline src/core/hle/service/apt_u.cpp]: Yes it was a typo in the code

**Comment by Subv** [comment]
@purpasmart96 Yes I am aware of #345, it doesn't signal the needed event, both PRs can co-exist nicely

**Comment by lioncash** [comment]
10/10 would force push

**Comment by yuriks** [comment]
LGTM I suppose

**Comment by archshift** [comment]
10/10 would LGTM again

File: src/core/hle/service/apt_u.cpp [hunk 2]
```diff
@@ -25,10 +25,12 @@ namespace APT_U {
 // correctly mapping it in Citra, however we still do not understand how the mapping is determined.
 static const VAddr SHARED_FONT_VADDR = 0x18000000;
 
-// Handle to shared memory region designated to for shared system font
+/// Handle to shared memory region designated to for shared system font
 static Handle shared_font_mem = 0;
 
 static Handle lock_handle = 0;
+static Handle notification_event_handle = 0; /// APT notification event handle
+static Handle pause_event_handle = 0; /// APT pause event handle
```
Comment (bunnei) [inline src/core/hle/service/apt_u.cpp]: Should be ///< If the comment appears after whatever it's describing.

**Comment by Subv** [comment]
Should be ready to merge @bunnei

**Comment by bunnei** [comment]
LGTM
